ethereumj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -457,7 +457,7 @@ public int databasePruneDepth() {
     @ValidateMe
     public List<Node> peerActive() {
         if (!config.hasPath("peer.active")) {
-            return Collections.EMPTY_LIST;
+            return Collections.emptyList();
```

Comment:
To avoid the warning
```
[Java] Type safety: The expression of type List needs unchecked conversion to conform to List<Node>
List EMPTY_LIST
The empty list (immutable). This list is serializable.
```
emptyList() also returns an immutable and serializable list